src/sentry/utils/yaml.py

```diff
@@ -0,0 +1,30 @@
+"""
+sentry.utils.yaml
+~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from functools import partial
+from yaml import load as _load, dump as _dump
+try:
+    # Try to load bindings with libyaml if available
+    from yaml import (
+        CLoader as Loader,
```

Comment:
You mean `load` vs `safe_load` ?

Unlikely. But I didn't want to alias `sentry.utils.yaml:load` as `safe_load` since that seemed wrong/misleading.